Assets/Scripts/UI/Pickup Pup/AdoptionScreen/PPAdoptionUIController.cs

```diff
@@ -1,6 +1,6 @@
 ﻿/*
  * Author: Isaiah Mann
- * Description: Controls the adoption UI
+ * Description: Controls the shelter screen
  */
 
 public class PPAdoptionUIController : PPUIController 
```

Comment:
Possibly worth renaming the class name to PPShelterUIController to ensure consistency of term